src/main/scala/com/monsanto/arch/cloudformation/model/resource/ElastiCache.scala

```diff
@@ -0,0 +1,64 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model.{ConditionRef, ResourceRef, Token}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+
+/**
+  * Cache security group.
+  *
+  * @param name
+  * @param Description
+  * @param SubnetIds
+  * @param Condition
+  */
+case class `AWS::ElastiCache::SubnetGroup`(
```

Comment:
Not a big deal but any reason not to include `CacheSubnetGroupName` from the [documentation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticache-subnetgroup.html)